hibernate-core/src/test/java/org/hibernate/orm/test/metamodel/hhh18868/HHH18868Test.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.orm.test.metamodel.hhh18868;
+
+import jakarta.persistence.metamodel.SingularAttribute;
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.JiraKey;
+import org.hibernate.testing.orm.junit.SessionFactory;
+import org.hibernate.testing.orm.junit.SessionFactoryScope;
+import org.junit.jupiter.api.Test;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DomainModel(annotatedClasses = {Summary.class, BaseSummary.class})
+@SessionFactory
+@JiraKey( "HHH-18858" )
+public class HHH18868Test {
```

Comment:
It would be nice to have a significant name for the test class, and place it in an appropriate package. How about `MappedSuperclassIdClassAttributesTest`, in the `org.hibernate.orm.test.idclass` package?

Reply:
Of course. I+ll do that.